packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -312,6 +331,16 @@ class SemanticsNode extends AbstractNode {
     }
   }
 
+  Set<SemanticsTag> _tags = new Set<SemanticsTag>();
+
+  /// Tag the [SemanticsNode] with [tag].
```

Comment:
I've filed https://github.com/flutter/flutter/issues/11716 to improve the docs on this for all three: tags, actions, flags.